fast/test/README.md

```diff
@@ -0,0 +1,66 @@
+# Using Locust to step through a Blue-Green Scenario
+
+## Install AS3 and FAST on the target BIG-IPs
+- Follow [these directions](https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/userguide/installation.html#installation) to install AS3  
+- Follow [these directions](https://clouddocs.f5.com/products/extensions/f5-appsvcs-templates/latest/userguide/install-uninstall.html) to install F5 Applications Services Templates (FAST)
+
+## Install the BlueGreen FAST Template
+- create a zip file of the `bluegreen.yml` file:
+
+```bash
+$ zip -r bluegreen.zip bluegreen.yaml
+```
+- Log into the target BIG-IP
+- Navigate to iApps -> Application Services -> Applications LX
+- Click on ```F5 Application Services Templates```
+- Click on the ```Templates``` tab
+- Click on ```Add Template Set``` 
+- Follow the subsequent steps using the ```bluegreen.zip``` file created above
+
+## Install Docker
+If you don't already have Docker installed, [install Docker](https://docs.docker.com/get-docker/)
+
+## Setup information for each of the VIPs
+Within the ```vips.py``` file, edit the VIPS_INFO array, adding a record for each VIP. The fields are *virtual address*, *AS3 Tenant Name*, and *AS3 Application Name*.
+
+```python
+VIP_INFO = [
+    ("192.0.2.10","Test","App"),
+    ("192.0.2.11","Test2","App")
+]
+```
+
+## Start a Locust Instance
+at a command line the following environment variables
```

Comment:
create? add?

Reply:
How about "at a command line, DEFINE the following environment variables"

Reply:
Whoops, I was too late. :)

Reply:
will do